docs/eventing/triggers/README.md

```diff
@@ -1,6 +1,6 @@
 A Trigger represents a desire to subscribe to events from a specific Broker.
 
-The `subscriber` value must be a [Destination](https://pkg.go.dev/knative.dev/pkg/apis/duck/v1#Destination).
+The `subscriber` value must be a Destination.
```

Comment:
In the other PR, I suggested a replacement for this.

Reply:
I have added the Destination yaml but my point is comment yaml the comment is mentioned for DeadLinkSink.
# DeadLetterSink is the sink receiving event that could not be sent to
# a destination.

So, is it still relevant to keep it?